crates/rolldown/src/stages/link_stage/mod.rs

```diff
@@ -300,7 +303,7 @@ impl<'a> LinkStage<'a> {
               let importee_linking_info = &self.metas[importee.idx];
               match rec.kind {
                 ImportKind::Import => {
-                  let is_reexport_all = importer.star_exports.contains(rec_id);
+                  let is_reexport_all = rec.meta.contains(ImportRecordMeta::IS_EXPORT_START);
```

Comment:
You could record this meta to get o(1) check here. However, it's better to keep the `star_exports` field for other parts.